test/unit/models/profileDiffs.test.js

```diff
@@ -0,0 +1,37 @@
+const chai = require("chai");
+const { expect } = chai;
+const cleanDb = require("../../utils/cleanDb");
+const profileDiffsQuery = require("../../../models/profileDiffs");
+const getProfileDiffs = require("../../fixtures/profileDiffs/profileDiffs");
+
+describe("profileDiffs", function () {
+  afterEach(async function () {
+    await cleanDb();
+  });
+
+  describe("fetchProfileDiffUnobfuscated", function () {
+    it("should successfully return profileDiffs", async function () {
+      const profileDiffs = getProfileDiffs();
+      const profileDiffId = await profileDiffsQuery.add(profileDiffs[1]);
+      const diff = await profileDiffsQuery.fetchProfileDiffUnobfuscated(profileDiffId);
+      expect(diff).haveOwnProperty("id");
+      expect(diff.profileDiffExists).to.equal(true);
+    });
+
```

Comment:
we should test the behavior when `profileDiffId` is `null`, `undefined`, or an empty string.